Source/SvgDocument.cs

```diff
@@ -349,9 +356,10 @@ public static T Open<T>(Stream stream, Dictionary<string, string> entities) wher
 
         private static T Open<T>(XmlReader reader) where T : SvgDocument, new()

```

Comment:
I am wondering whether we should not also allow the flag to be set on a "per-instance" case, e.g. in the constructor or having a specialized inheritor for this. For now the flag will be fine, but if you use multiple callees of the library you might encounter strange behavior (if you for example use the grapics but also have a component that will set the flag to false this might break the rendering part). Those specific situations are rare (and require you to have one or more sub libraries that use the flag/rendering) and not realy likely, but having system-wide flags might be a problem in the very far future.

Reply:
`SvgDocument` is created using `SvgElementFactory` so having per instance property would require setting the value of `SkipGdiPlusCapabilityCheck` when creating documents in factory. And you can have documents inside document and referenced external documents also created via factory.

Reply:
@gvheertum - I agree with @wieslawsoltes that this would be difficult, and that the current state is sufficient. I will merge as is if you can live with that...

Reply:
I totally agree :) Having to throw the flag all over the place is indeed not the way to go now! So I can totally live with it.